entity/columns_json.go

```diff
@@ -0,0 +1,84 @@
+package entity
+
+import (
+	"fmt"
+
+	"github.com/cockroachdb/errors"
+	schema "github.com/milvus-io/milvus-proto/go-api/schemapb"
+)
+
+var _ (Column) = (*ColumnJSONBytes)(nil)
+
+// ColumnJSONBytes column type for JSON.
+// all items are marshaled json bytes.
+type ColumnJSONBytes struct {
```

Comment:
Why not just `ColumnJSON`?

Reply:
Will provided ColumnJSON with entity objects as values